pantheon-files-daemon/FileManager1.vala

```diff
@@ -43,7 +43,7 @@ public class FileManager1 : Object {
         string cmd = "pantheon-files -t";
 
         foreach (string s in uris) {
-            cmd += (" " + s);
+            cmd += (" \"" + PF.FileUtils.sanitize_path (s) + "\"");
```

Comment:
I believe this should actually be single quotes based upon the bash docs:
https://www.gnu.org/software/bash/manual/html_node/Single-Quotes.html#Single-Quotes
https://www.gnu.org/software/bash/manual/html_node/Double-Quotes.html#Double-Quotes

> Enclosing characters in double quotes (‘"’) preserves the literal value of all characters within the quotes, with the exception of ‘$’, ‘`’, ‘\’, and, when history expansion is enabled, ‘!’.

> Enclosing characters in single quotes (‘'’) preserves the literal value of each character within the quotes.

Again, I still haven't tested this yet, but it looks as though there's still a risk of bash interpreting some special characters if they happen to be in the path string.